packages/content/src/content.ts

```diff
@@ -1,8 +1,292 @@
 /* Copyright (c) 2018 Environmental Systems Research Institute, Inc.
  * Apache-2.0 */
 
+import { ISpatialReference, IExtent } from "@esri/arcgis-rest-types";
+import { IItem, getItem } from "@esri/arcgis-rest-portal";
 import { request } from "@esri/arcgis-rest-request";
+import {
+  HubType, IHubContent, IHubGeography, IHubRequestOptions, IBBox,
+  getType, getCategory, getItemThumbnailUrl, getPortalApiUrl
+} from "@esri/hub-common";
 
+function buildUrl (base: string, path: string): string {
+  const baseUrl = base.endsWith('/') ? base : `${base}/`;
+  const url = new URL(path, baseUrl);
+  return url.toString();
+}
+
+// TODO: make this real; use request() under the hood?
+export function hubRequest (url: string, requestOptions?: IHubRequestOptions) {
+  return fetch(url, {
```

Comment:
I know this is a bigger change, but if we're to be isomorphic, it'd be nice if we didn't require fetch.

Reply:
I think we can get this from https://esri.github.io/hub.js/api/common/IHubRequestOptions/#fetch the way rest-js does